src/gleam/bit_array.gleam

```diff
@@ -155,3 +157,56 @@ pub fn base16_encode(input: BitArray) -> String
 @external(erlang, "gleam_stdlib", "base16_decode")
 @external(javascript, "../gleam_stdlib.mjs", "base16_decode")
 pub fn base16_decode(input: String) -> Result(BitArray, Nil)
+
+/// Converts a bit array to a string containing the decimal value of each byte.
+///
+/// ## Examples
+///
+/// ```gleam
+/// inspect(<<0, 20, 0x20, 255>>)
+/// // -> "<<0, 20, 32, 255>>"
+///
+/// inspect(<<100, 5:3>>)
+/// // -> "<<100, 5:size(3)>>"
+/// ```
+/// 
+pub fn inspect(input: BitArray) -> String {
```

Comment:
This looks great, thank you!

Sorry, one last thing. Could you use a string accumulator throughout and append to that? It will be much faster than using a string builder in this case.

```gleam
"<<"
|> do_inspect(input, _)
|> string.append(">>")
```

Reply:
Done!